src/screens/EditTextScreen.tsx

```diff
@@ -5,15 +5,32 @@ import { useNavigationParam } from 'react-navigation-hooks';
 
 import { Header, InputItem, Button, ScreenTemplate } from 'app/components';
 
+type KeyboardType =
+  | 'default'
+  | 'email-address'
+  | 'numeric'
+  | 'phone-pad'
+  | 'number-pad'
+  | 'decimal-pad'
+  | 'visible-password'
+  | 'ascii-capable'
+  | 'numbers-and-punctuation'
+  | 'url'
+  | 'name-phone-pad'
+  | 'twitter'
+  | 'web-search'
+  | undefined;
 export interface EditTextProps {
   title: string;
   onSave: (value: string) => void;
   label: string;
   value?: string;
+  keyboardType?: KeyboardType;
```

Comment:
isn't `KeyboardType` exported from `react-native`?

Reply:
oh, you are right